demos/inputs.html

```diff
@@ -0,0 +1,295 @@
+<!doctype html>
+<html>
+	<head>
+		<meta charset="utf-8">
+		<title>CSS Chassis - Inputs</title>
+		<meta name="description" content="Button skeleton for styling">
+		<meta name="viewport" content="width=device-width, initial-scale=1">
+		<link rel="stylesheet" href="../dist/css/chassis.css">
+		<link rel="stylesheet" href="demos.css">
+	</head>
+	<body>
+
+		<h2>Inputs - (Size: Default)</h2>
+
+		<h3>Checkboxes</h3>
+		<!-- Custom check boxes -->
+
+		<input id="check1" type="checkbox" class="checkbox">
+		<label for="check1" class="checkbox-label">Choose this</label>
+
+		<input id="check2" type="checkbox" class="checkbox">
+		<label for="check2" class="checkbox-label">And this</label>
+
+		<input id="check3" type="checkbox" class="checkbox">
+		<label for="check3" class="checkbox-label">And this too</label>
+
+		<!-- Custom Radio button -->
+
+		<h3>Radio buttons</h3>
+
+		<div>
+			<input id="radio-1" class="radiobutton" name="radio-group" type="radio" checked>
+			<label for="radio-1" class="radiobutton-label">Second Choice</label>
+		</div>
+
+		<div>
+			<input id="radio-2" class="radiobutton" name="radio-group" type="radio" checked>
+			<label for="radio-2" class="radiobutton-label">Third Choice</label>
+		</div>
+
+		<div>
+			<input id="radio-3" class="radiobutton" name="radio-group" type="radio" checked>
+			<label for="radio-3" class="radiobutton-label">Fourth Choice</label>
+		</div>
+
+		<div>
+			<input id="radio-4" class="radiobutton" name="radio-group" type="radio" checked>
+			<label for="radio-4" class="radiobutton-label">Fifth Choice</label>
+		</div>
+
+
+		<label class="input-label">Name: </label>
+		<input class="input" type="text" placeholder="Enter your Name">
+		<label class="input-label">Password: </label>
+		<input class="input" type="password" placeholder="Choose a password">
+		<!-- 	<label class="input-label">Color: </label>
```

Comment:
Color, date e.t.c inputs are not supported in many browsers. Uncommenting them would upset Travis